piet-gpu/shader/kernel4.comp

```diff
@@ -201,15 +201,16 @@ void main() {
             break;
         case Cmd_BeginClip:
             for (uint k = 0; k < CHUNK; k++) {
-                blend_stack[blend_sp][k] = packsRGB(vec4(rgb[k], clamp(abs(area[k]), 0.0, 1.0)));
+	    	// Safeguard against overflow, but do not guarantee correct behavior.
```

Comment:
This may be ok to do here, but I'll note that piet-gpu doesn't have many other of these runtime checks or guards. Perhaps this check should go in the CPU driver, where the user can get a proper error about the limit.

Reply:
I added an additional check in render_ctx.